### ISSUE TITLE
Plugin ideas (for later)

### ISSUE THREAD

**Issue by jadjoubran** [jadjoubran/webdash#57]
Feel free to suggest new plug-in ideas here

- [ ] Git plug-in for webdash (branches + versions)
- [ ] Google analytics realtime embed (can't be implemented yet because <google-analytics> only works on Polymer 1)

**Comment by jadjoubran** [comment]
I got this recommendation yesterday when I was presenting webdash to a group..
I feel like this would creep on Chrome dev tools' turf
If there's anything that can be improved in Lighthouse or Dev tools + lighthouse, we can already contribute there
But anyway the plugin system is completely open, so if anyone feels the need to do that, then that's perfectly fine

It would also be cool to have a something built on top of lighthouse, something like a nice graph from @calibreapp
I'll ping them on twitter and see if they're interested

**Comment by benschwarz** [comment]
Hey @jadjoubran, 

Thanks for reaching out on Twitter (I'm the founder of Calibre). 

We have a new command line interface / api that should make it trivial to get stats out of an account. I'd be happy to provide a test account for you to use.

To get started: 

* Register for a free trial, then ping hello @ calibreapp.com with the email address that you used to sign up.
* I'll boost your trial period to be far longer and share some details about how we can get the stats from Calibre.
* From the Calibre side, we can provide a documentation page (say Calibreapp.com/webdash) to help people find it, and a docs page to help folks get started. 

How's all that sound?

**Comment by jadjoubran** [comment]
Thanks @benschwarz 
sounds good! Will start working on this as soon as webdash v1.3 is released
So I'll only signup for an account once I start working on the plugin (tracked in #58, if you'd like to get updates, please watch the other issue thanks!)

I'm also thinking of opening up sponsorships in the future in order to sustain webdash, I'll let you know about it then if you're interested
Cheers

**Comment by jadjoubran** [comment]
- [ ] IFTTT plugin using [node-ifttt-maker](https://www.npmjs.com/package/node-ifttt-maker)

Would love to have someone with IFTTT API experience

**Comment by cjsaylor** [comment]
Would be cool for a react-scripts like API proxy to show requests as you make them with response stats.

**Comment by jadjoubran** [comment]
@cjsaylor can you give more explanation please?

**Comment by bardiarastin** [comment]
what about CI status plugin ?

**Comment by jadjoubran** [comment]
that would be cool! We'd have to build one for CircleCi and another for Travis right?

**Comment by bardiarastin** [comment]
yes they are the most popular ones, later we can add support for private CI's 👍

**Comment by jadjoubran** [comment]
I was thinking whether it makes sense to create a plugin that simply shows status/building or error but then this can be easily embedded in your README.md and will show up if you use `webdash-readme-preview`
so if we were to build a plugin for CIs, then it make sense to show the last 2-3 builds + with a bit more info

**Comment by jadjoubran** [comment]
Hey @benschwarz 
I haven't heard back from you after I sent the email
I'm planning on working on the calibre plugin soon
Cheers!

**Comment by jadjoubran** [comment]
Thansk @MartynKeigher 
Would you be interested in creating a plugin? It's quite easy and I could help
